py3status/modules/sysdata.py

```diff
@@ -357,7 +361,7 @@ def _calc_cpu_percent(self, cpu):
         )
         return used_percent
 
-    def _get_cputemp_with_lmsensors(self, zone=None):
+    def _get_cputemp_with_lmsensors(self, zone=None, zone_sensor=None):
```

Comment:
since it has a default value, I think we should not make `zone_sensor` a keyword argument with a default value

why not accessing it directly as self.zone_sensor below?

this would avoid having to pass it everywhere else